src/stylesheets/components/_sidenav.scss

```diff
@@ -1,8 +1,28 @@
 
 .usa-sidenav-list {
   @include usa-sidenav-list;
+  @include margin(null (-$site-margins-mobile) null (-$site-margins-mobile));
+  border-bottom: 1px solid;
+  border-top: 1px solid;
```

Comment:
In the future, is the top and bottom borders something we'd like to do in desktop as well or only mobile?